crawl-ref/source/actor.cc

```diff
@@ -330,6 +330,7 @@ int actor::spirit_shield(bool calc_unid, bool items) const
 
     if (is_player())
         ss += you.get_mutation_level(MUT_MANA_SHIELD);
+        ss += have_passive(passive_t::ely_guardian_spirit);
```

Comment:
Seems like misleading indentation.
Did you mean to put this in the `is_player()` branch? If so, add `{ ... }`
If not, it should be indented with the code outside the branch

Reply:
Um, I'm sorry, but can you tell me in more detail? I didn't understand well due to lack of skills.

Reply:
The way you wrote the code, it looks like as if you meant to write
```
    if (is_player())
    {
        ss += you.get_mutation_level(MUT_MANA_SHIELD);
        ss += have_passive(passive_t::ely_guardian_spirit);
    }
```
That is, the `ss` value would only be added if the actor is the player.

But what you actually wrote is equivalent to 
```
if(is_player())
{
  ss += you.get_mutation_level(MUT_MANA_SHIELD);
}
ss += have_passive(passive_t::ely_guardian_spirit);
```
That is, the check is made for all actors. Notice the line with Ely's guardian spirit is not indented.

The fact that it looks like one thing but is actually something else is why I called it "misleading". I didn't call it wrong, because I don't think it would have an effect on any actor other than the player. It's still an issue for future maintainers, who might be confused about the intent.

Reply:
oh, ok. I'm I understood. Thank you. :)